src/Nethermind/Nethermind.Core/Crypto/Signature.cs

```diff
@@ -52,26 +57,27 @@ public Signature(string hexString)
             : this(Core.Extensions.Bytes.FromHexString(hexString))
         {
         }
+        public Span<byte> Bytes => MemoryMarshal.AsBytes(MemoryMarshal.CreateSpan(ref _signature, 1));
+        public override Memory<byte> Memory => CreateMemory(64);
```

Comment:
This doesn't allocate but return parts of binary representation of the class as `Memory<byte>`?

Reply:
Yeah, class uses itself as the backing by inheriting from `MemoryManager<byte>`; then Memory when Span is called invokes the Span method. 

Is a bit weird and like a sledgehammer for a nut, but skips all the allocations 🙂

Reply:
Shouldn't we store it somewhere or is it better to create it on every call?

Reply:
Easy enough to create it; otherwise it will expand `Signature` itself by 16 bytes